app/Controllers/Course.hs

```diff
@@ -23,7 +23,8 @@ retrieveCourse = do
 -- | Builds a list of all course codes in the database.
 index :: ServerPart Response
 index = do
-  response <- liftIO $ runSqlite databasePath $ do
+  dbPath <- liftIO databasePath
```

Comment:
Okay, you've done this correctly (👍) but looking at the breadth of changes, I'd like you to go a step further and create a new definition in `Config.hs` that encapsulates this repeated code (lines 26 and 27).

Based on the documentation for [runSqlite](https://hackage.haskell.org/package/persistent-sqlite-2.13.3.0/docs/Database-Persist-Sqlite.html#v:runSqlite), maybe something like `runDb :: MonadUnliftIO m	=> ReaderT SqlBackend (NoLoggingT (ResourceT m)) a -> m a`.

To be explicit, this function should load `databasePath` and then call `runSqlite` on that plus whatever its argument is.